tools/accuracy_checker/configs/person-detection-0100.yml

```
@@ -0,0 +1,48 @@
+models:
+  - name: person-detection-0100
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   intel/person-detection-0100/FP32/person-detection-0100.xml
+        weights: intel/person-detection-0100/FP32/person-detection-0100.bin
+        adapter: ssd
+
+      - framework: dlsdk
+        tags:
+          - FP16
+        model:   intel/person-detection-0100/FP16/person-detection-0100.xml
+        weights: intel/person-detection-0100/FP16/person-detection-0100.bin
+        adapter: ssd
+
+    datasets:
+      - name: person_detection
+
+        preprocessing:
+          - type: resize
+            dst_width: 256
+            dst_height: 256
+
+        postprocessing:
+          - type: resize_prediction_boxes
+          - type: filter
+            apply_to: annotation
+            height_range: 100
+            aspect_ratio: 0.666, 5
+            is_empty: True
+            min_visibility: partially occludedsssssssssssssssssssss
```

```suggestion
            min_visibility: partially occludedsssssssssssssssssssss

```suggestion
            min_visibility: partially occluded
```